### PR TITLE
Fix bottom padding for alert dialogs when supress is null

### LINKED ISSUE #188: 🐛 Remove extra padding in alert dialog
**Description**

This one is a minor UI glitch, but I don't have time to raise a PR at the moment (it'd be a good one for any new contributors)... when the `suppress` flag is set to `null` in the alert dialog, we end up with two `SizedBox` widgets in a row (which increases the padding at the bottom under the button).

image

This first `SizedBox` should be included in the `if` condition [here](https://github.com/GroovinChip/macos_ui/blob/e0ebd3babf36d5f7382dc0a4a219cb685d6273af/lib/src/dialogs/macos_alert_dialog.dart#L229)

ie. it should look something like (untested):
```
if (suppress != null) ...[
  const SizedBox(height: 16),
  DefaultTextStyle(
    style: MacosTheme.of(context).typography.headline,
    child: suppress!,
  ),
],
const SizedBox(height: 16),
```

**Steps To Reproduce**

Show a `MacosAlertDialog` where `suppress` is set to `null`

### PR DESCRIPTION
**Pull request by Andrewngabriel**
Fix padding on `MacosAlertDialog` when `supress` field is null.

Fixes #188 

## Pre-launch Checklist

- [x] I have run `dartfmt` on all changed files 
- [x] I have incremented the package version as appropriate and updated `CHANGELOG.md` with my changes 
- [x] I have added/updated relevant documentation 
- [x] I have run "optimize/organize imports" on all changed files
- [x] I have addressed all analyzer warnings as best I could

### CODE REVIEW

**Comment by GroovinChip** [comment]
Hi @Andrewngabriel, thanks for your PR! I just tried to test your changes and was met with some build errors. Are you seeing this as well?
```
error: Unable to load contents of file list: '/Target Support Files/Pods-Runner/Pods-Runner-frameworks-Debug-input-files.xcfilelist' (in target 'Runner' from project 'Runner')
error: Unable to load contents of file list: '/Target Support Files/Pods-Runner/Pods-Runner-frameworks-Debug-output-files.xcfilelist' (in target 'Runner' from project 'Runner')
```

It's possible that the `macos` runner needs to be regenerated, I do know that it has been changed since I created it. I'll try that.

EDIT: regenerating the `macos` runner fixed the problem in the `dev` branch. @Andrewngabriel please pull in the latest changes from dev and I'll test your changes.

**Comment by Andrewngabriel** [comment]
@GroovinChip I ran into the same error but after running `pod install` from inside the macos directory, it fixed the build issue and was able to run ok.

I did notice however that `Podfile.lock` was updated. Should I commit that as well?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [0.12.3+1]
```
Comment (GroovinChip) [inline CHANGELOG.md:1]: Please update the version in the `pubspec.yaml` to match this.

Comment (Andrewngabriel) [reply CHANGELOG.md:1]: Done

**Comment by GroovinChip** [comment]
> I did notice however that `Podfile.lock` was updated. Should I commit that as well?

Check the various `.gitignore` files and see if it is currently being tracked. If it is, commit it. If not, do not commit it.

**Comment by GroovinChip** [comment]
@Andrewngabriel don't forget to run `flutter pub get` at the package level after update the package-level `pubspec.yaml` since the example's `pubspec.lock` is tracked; if you don't, the lock file won't update.

**Comment by Andrewngabriel** [comment]
I just tried running after latest changes and it seems like there is no longer a Podfile.lock in the macos directory. Not sure if it was removed but the app is building and running fine. Let me know if you're still getting the same error

**Comment by GroovinChip** [comment]
> I just tried running after latest changes and it seems like there is no longer a Podfile.lock in the macos directory. Not sure if it was removed but the app is building and running fine. Let me know if you're still getting the same error

Nah mate, my error was resolved once I regenerated the macos runner. 

Anyway, this all LGTM to me. Waiting on checks and then I'll merge.

**Review by GroovinChip — APPROVED** [review]
LGTM

**Comment by GroovinChip** [comment]
Congratulations on your first merged PR to macos_ui, @Andrewngabriel!!! Thank you so much for contributing!

**Comment by Andrewngabriel** [comment]
@GroovinChip Thank you! Expect many more to come 🙂

**Comment by GroovinChip** [comment]
@Andrewngabriel hurray!!! Please let me know what else you'd like to work on by assigning yourself an issue and tagging me on it, and if you'd like to join the contributors Slack group please send me your email address and I'll invite you! I'll also add you as a collaborator to this repo so that you don't have to submit PR's from your fork - you'll be able to do it from this repo.

**Comment by Andrewngabriel** [comment]
@GroovinChip That would be amazing! I just DMed you my email on Twitter

**Pull request merged**